src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1567,6 +1568,16 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
             }
         }
 
+        if ([] !== $config['globals']) {
+            $container
+                ->register('translator.default_parameters', DefaultParametersTranslator::class)
+                ->setDecoratedService('translator', null, -1) // Lower priority than "translator.data_collector"
```

Comment:
do we need a specific priority compared to the PseudoLocalizationTranslator ?

Reply:
IIUC the priority meaning, I guess we should put `DefaultParametersTranslator` before `PseudoLocalizationTranslator`, in order to leverage the pseudo-localization for replaced values of defaults parameters. WDYT?

Reply:
> IIUC the priority meaning, I guess we should put `DefaultParametersTranslator` before `PseudoLocalizationTranslator`, in order to leverage the pseudo-localization for replaced values of defaults parameters. WDYT?

Not really because `PseudoLocalizationTranslator` calls the decorated stack before updating the translated text. Also it does not rely on parameters, which are the only thing changed by `DefaultParametersTranslator`.

See https://github.com/symfony/symfony/blob/9519b54417c09c49496a4a6be238e63be9a73465/src/Symfony/Component/Translation/PseudoLocalizationTranslator.php#L91